protocol/generated.proto

```diff
@@ -12,7 +12,9 @@ option go_package = "protocol";
 message ParseUASTRequest {
 	option (gogoproto.goproto_getters) = false;
 	option (gogoproto.typedecl) = false;
-	string content = 1;
+	string filename = 1;
+	string language = 2;
```

Comment:
If the user wants to override the language version (for example python 2 vs 3) will this version be embedded in the language? (like in "python2") or should we add a new "version' optional field?